pkg/utils/key.go

```diff
@@ -0,0 +1,70 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/spf13/pflag"
+)
+
+// ParseKey parse key by given format
+func ParseKey(flags *pflag.FlagSet, key string) (string, error) {
```

Comment:
An key is better to be represented by `[]byte` rather than a string. After parsing a string, you'd better return a `[]byte` here.

Reply:
Sure, update [it](https://github.com/pingcap/br/pull/101/files#diff-83d7917e5fba66e42677eb2d93c47ed5R15)